packages/alfa-style/src/property/helpers/mask-layers.ts

```diff
@@ -0,0 +1,38 @@
+import { List, type Value } from "@siteimprove/alfa-css";
+
+import type { Style } from "../../style.js";
+
+/**
+ * {@link https://drafts.fxtf.org/css-masking/#layering}.
+ *
+ * @remarks
+ * The computed value depends on the number of layers.
+ * A layer is created for each of the comma separated values for `mask-image`.
+ *
+ * If there are more values than layers, the excess values are discarded.
+ * Otherwise, the values must be repeated until the number of values matches the number of layers.
+ */
+export function matchLayers<V extends Value>(
```

Comment:
**Suggestion:** Given that the `Style` is only used to compute a number, this could take that number directly, and be moved as `List#extendsToLength` or something similar. That name would match the effect (rather than the intention), and the method would belong to the data structure it changes, improving cohesion.

`List#extendsToLength` could also take another `List` as parameter and get the length itself (instead or in addition to taking a number).